src/pages/registration/DescriptionPanel.tsx

```diff
@@ -37,27 +38,55 @@ export const DescriptionPanel = () => {
 
   return (
     <InputContainerBox>
-      <Field name={DescriptionFieldNames.Title}>
-        {({ field, meta: { touched, error } }: FieldProps<string>) => (
-          <TextField
-            {...field}
-            id={field.name}
-            value={field.value ?? ''}
-            required
-            data-testid={dataTestId.registrationWizard.description.titleField}
-            variant="filled"
-            fullWidth
-            label={t('common.title')}
-            error={touched && !!error}
-            helperText={<ErrorMessage name={field.name} />}
-          />
-        )}
-      </Field>
+      <Box
+        sx={{
+          display: 'grid',
+          rowGap: '1rem',
+          columnGap: '0.5rem',
+          gridTemplateColumns: '1fr auto',
+        }}>
+        <Field name={DescriptionFieldNames.Title}>
+          {({ field, meta: { touched, error } }: FieldProps<string>) => (
+            <TextField
+              {...field}
+              value={field.value ?? ''}
+              required
+              data-testid={dataTestId.registrationWizard.description.titleField}
+              variant="filled"
+              fullWidth
+              label={t('common.title')}
+              error={touched && !!error}
+              helperText={<ErrorMessage name={field.name} />}
+            />
+          )}
+        </Field>
+        <Field name={`${DescriptionFieldNames.AlternativeTitles}.und`}>
+          {({ field }: FieldProps<string>) => (
+            <>
+              <Button
+                startIcon={<AddCircleOutlineIcon />}
```

Comment:
kanskje sette en maks-høyde for å unngå at knappen blir veldig stor?
```suggestion
                sx={{ height: 'fit-content' }}
                startIcon={<AddCircleOutlineIcon />}
```
![bilde](https://user-images.githubusercontent.com/6313468/222422641-5e1c3288-2f7e-491a-be6e-1f1d022ebbf2.png)